Assets/Scripts/Characters/Player/Inventory.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Inventory : MonoBehaviour
+{
+    private List<ItemBase> items;
+
+    private KeyCode showInventory = KeyCode.I;
+    private KeyCode collectItems = KeyCode.F;
+
+    private GameObject cellContainer;
+    private GameObject Canvas;
+    public GameObject cellContainerPrefab;
+
+    private CollectableItems CollectableItems;
```

Comment:
```suggestion
    private CollectableItems collectableItems;
```
Не очень хорошо когда совпадают имена классов с именами переменных, да и приватные переменные обычно с маленькой буквы пишут